conf/layer.conf

```diff
@@ -26,6 +26,9 @@ LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-plugins-ugly"
 LICENSE_FLAGS_WHITELIST += "commercial_libomxil"
 LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-omx"
 
+# WPE TVControl on rpi needs libav plugin
+LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-libav"
```

Comment:
this is probably not right. Since we will force indirectly someone to accept a commercial license without them knowing about it. Instead please document it in README.